tests/test_dunders.py

```diff
@@ -200,6 +200,13 @@ class C(object):
 
         assert "C(_x=42)" == repr(i)
 
+    def test__repr_uninitialized_member(self):
```

Comment:
This has a superfluous underscore and needs a [test docstring](https://jml.io/pages/test-docstrings.html).